Update JEDI hashes (20250512)

RussTreadon-NOAA opened the pull request:
# Description

Weekly update of JEDI hashes.  

The following changes are also included in this PR
- `ci/driver.sh` - make test check case insensitive
- `test/gw-ci/create_exp.sh` - update path to `gw_setup.sh`
- `test/snow/CMakeLists.txt` - remove `test_gdasapp_snow_letkfoi_snowda` and supporting yamls

# Companion PRs

none

# Issues

Resolves #1644 
Resolves #1645


# Automated CI tests to run in Global Workflow
- [x] atm_jjob <!-- JEDI atm single cycle DA !-->
- [x] C96C48_ufs_hybatmDA <!-- JEDI atm cycled DA !-->
- [x] C96C48_hybatmaerosnowDA  <!-- JEDI aero/snow cycled DA !-->
- [x] C48mx500_3DVarAOWCDA <!-- JEDI low-res marine 3DVar cycled DA  !-->
- [x] C48mx500_hybAOWCDA <!-- JEDI marine hybrid envar cycled DA !-->
- [x] C96C48_hybatmDA <!-- GSI atm cycled DA !-->

RussTreadon-NOAA committed:
remove test_gdasapp_snow_letkfoi_snowda (#1660)

RussTreadon-NOAA committed:
Update to new stable build on 20250506

RussTreadon-NOAA committed:
update da-utils hash

RussTreadon-NOAA committed:
Merge branch 'develop' into feature/stable-nightly

RussTreadon-NOAA committed:
make driver.sh automated ci check case independent

RussTreadon-NOAA committed:
Merge branch 'develop' into feature/stable-nightly

RussTreadon-NOAA committed:
update jcb-gdas hash to bring in gsibec interpolation changes (#1610)

RussTreadon-NOAA committed:
update path to gw_setup.sh for gw-ci (#1670)

RussTreadon-NOAA committed:
Update to new stable build on 20250507

RussTreadon-NOAA committed:
Merge branch 'develop' into feature/stable-nightly

RussTreadon-NOAA committed:
Merge branch 'develop' into feature/stable-nightly

RussTreadon-NOAA committed:
Update to new stable build on 20250509

RussTreadon-NOAA committed:
add gdas_waveinit dependency for C48mx500_3DVarAOWCDA gdas_fcst

DavidNew-NOAA commented:
@RussTreadon-NOAA What's the status of this PR? Is it ready to be merged soon?

DavidNew-NOAA commented:
Oh yes, I forgot, COM changes in GDASApp have broken the workflow. I will test in my GW PR branch

RussTreadon-NOAA commented:
@DavidNew-NOAA 

I modified the Hera stable-nightly job to clone the g-w PR [#3624](https://github.com/NOAA-EMC/global-workflow/pull/3624) branch and replaced `sorc/gdas.cd` with `feature/stable-nightly`.  This combination works as today's (5/9) run reported.
```
Automated GW-GDASApp Testing Results:
Machine: hera

Start: Fri May  9 00:43:19 UTC 2025 on hfe08
---------------------------------------------------
Build:                                 *SUCCESS*
Build: Completed at Fri May  9 01:35:58 UTC 2025
---------------------------------------------------
Tests: ctest -j12 -R gdasapp
Tests:                                 *SUCCESS*
Tests: Completed at Fri May  9 04:23:44 UTC 2025
Tests: 100% tests passed, 0 tests failed out of 135
```

If we are comfortable doing so we can merge this PR into GDASApp `develop`.  Then we can update the `sorc/gdas.cd` hash in g-w PR [#3624](https://github.com/NOAA-EMC/global-workflow/pull/3624).   

That's one piece of the GSIBEC puzzle.  The other part is the need to update `gdas_gsibec_ver` in g-w `versions/fix.ver`.  

g-w PR [#3676](https://github.com/NOAA-EMC/global-workflow/pull/3676) does this.  The `gdas_gsibec_ver` change in PR [#3676](https://github.com/NOAA-EMC/global-workflow/pull/3676) could be added to g-w PR [#3624](https://github.com/NOAA-EMC/global-workflow/pull/3624).   

The resulting g-w PR [#3624](https://github.com/NOAA-EMC/global-workflow/pull/3624) should pass g-w CI and GDASApp ctests.

Do we start this process now or wait until Monday (5/12) morning?

DavidNew-NOAA commented:
Let's just merge this PR now and I'll add the version change to my PR + update gdas hash

RussTreadon-NOAA commented:
Thanks @DavidNew-NOAA
